Sources/SwiftOracle/BindVar.swift

```diff
@@ -18,7 +18,7 @@ public class BindVar: StringLiteralConvertible, IntegerLiteralConvertible, Boole
         
     }
     public init (_ value: String) {
-        var v = Array(value.nulTerminatedUTF8).map( {Int8(bitPattern: $0) })
+        var v = Array(value.utf8CString)
```

Comment:
after correcting the above error, i encountered the following error while the connection IP address is wrong, So "nil" value need to be  fixed:
fatal error: unexpectedly found nil while unwrapping an Optional value
Current stack trace:
0    libswiftCore.so                    0x00007fc7446f4300 swift_reportError + 117
1    libswiftCore.so                    0x00007fc744705750 _swift_stdlib_reportFatalError + 61
2    libswiftCore.so                    0x00007fc7445151c3 <unavailable> + 0
3    libswiftCore.so                    0x00007fc74467028d <unavailable> + 0
4    libswiftCore.so                    0x00007fc7445151c3 <unavailable> + 0
5    libswiftCore.so                    0x00007fc744634290 specialized _fatalErrorMessage(StaticString, StaticString, StaticString, UInt, flags : UInt32) -> Never + 96
6    for_orcl_driver                    0x0000000000405b51 <unavailable> + 0
7    for_orcl_driver                    0x0000000000405cf8 <unavailable> + 0
8    for_orcl_driver                    0x0000000000405d88 <unavailable> + 0
9    libswiftCore.so                    0x00007fc7445927b0 specialized _print_unlocked<A, B where ...> (A, inout B) -> () + 1173
10   libswiftCore.so                    0x00007fc744669e23 <unavailable> + 0
11   libswiftCore.so                    0x00007fc744669bd0 specialized specialized _print<A where ...> ([Any], separator : String, terminator : String, to : inout A) -> () + 42
12   libswiftCore.so                    0x00007fc74466a26f <unavailable> + 0
13   libswiftCore.so                    0x00007fc744669c30 specialized print([Any], separator : String, terminator : String) -> () + 45
14   libswiftCore.so                    0x00007fc744596eb0 print([Any], separator : String, terminator : String) -> () + 29
15   for_orcl_driver                    0x000000000040609c <unavailable> + 0
16   for_orcl_driver                    0x00000000004082e1 <unavailable> + 0
17   for_orcl_driver                    0x00000000004082f9 <unavailable> + 0
18   libocilib.so.4                     0x00007fc7441ae700 OCI_ExceptionRaise + 34
19   libocilib.so.4                     0x00007fc7441ae730 OCI_ExceptionOCI + 182
20   libocilib.so.4                     0x00007fc7441a8aa0 OCI_ConnectionAttach + 303
21   libocilib.so.4                     0x00007fc7441abac0 OCI_ConnectionCreateInternal + 30
22   libocilib.so.4                     0x00007fc7441abb20 OCI_ConnectionCreate + 132
23   for_orcl_driver                    0x00000000004084a2 <unavailable> + 0
24   for_orcl_driver                    0x00000000004055d5 <unavailable> + 0
25   libc.so.6                          0x00007fc742f209d0 __libc_start_main + 240
26   for_orcl_driver                    0x0000000000404f19 <unavailable> + 0
Illegal instruction

----------------------------------------------------------------------------------
When i entered the correct host address and all the other correct information then i got:
lip@ubuntu:~/for_orcl_driver$ .build/debug/for_orcl_driver
SwiftOracle.Row
SwiftOracle.Row
2

instead of showing two records